packages/genymotion-bin/genymotion-bin.pacscript

```diff
@@ -15,4 +15,5 @@ maintainer="Arrowsome <ramtintoosi@gmail.com>"
 package() {
   sudo chmod +x "./genymotion-${pkgver}-linux_x64.bin"
   sudo "./genymotion-${pkgver}-linux_x64.bin" -y -d "${pkgdir}/opt"
+  sudo sed -i "s|${pkgdir}||g" /usr/share/applications/genymobile-genymotion.desktop
```

Comment:
this should have `${pkgdir}` before the path to the desktop.

Reply:
In that case: 
```shell
sudo sed -i "s|${pkgdir}||g" "${pkgdir}/usr/share/applications/genymobile-genymotion.desktop"`
```
_sed: can't read /usr/src/pacstall/genymotion-bin/usr/share/applications/genymobile-genymotion.desktop: No such file or directory_

Reply:
Why are you installing a file straight to root? Everything goes in `${pkgdir}`.